toolchain/lower/handle_aggregates.cpp

```diff
@@ -141,9 +142,12 @@ static auto EmitAggregateInitializer(FunctionContext& context,
       // non-constant initialization.
       for (auto [i, ref_id] : llvm::enumerate(refs)) {
         if (context.sem_ir().constant_values().Get(ref_id).is_constant()) {
-          auto init =
-              context.sem_ir().insts().GetAs<SemIR::InitializeFrom>(ref_id);
-          HandleInst(context, ref_id, init);
```

Comment:
Does HandleInst(... SemIR::InitializeFrom) still have callers, now that we're somewhat doing its job here inline?

Reply:
Yeah, I think if the `InitializeFrom` is non-constant, then `HandleInst(... SemIR::InitializeFrom)` gets called from the direct emission path here:
```
Carbon::Lower::HandleInst(Carbon::Lower::FunctionContext&, Carbon::SemIR::InstId, Carbon::SemIR::InitializeFrom) ()
    at toolchain/lower/handle.cpp:182
LowerInstHelper<Carbon::SemIR::InitializeFrom> () at toolchain/lower/function_context.cpp:86
LowerInst () at ./toolchain/sem_ir/inst_kind.def:80
LowerBlockContents () at toolchain/lower/function_context.cpp:56
```